docs/usage/templates.md

```diff
@@ -15,10 +15,16 @@ Some are configuration options passed through, while others are generated as par
 
 ## Exposed config options
 
+<!-- Autogenerate in https://github.com/renovatebot/renovatebot.github.io -->
```

Comment:
This comment refers to autogenerated files in the `renovatebot.github.io` repository, but I think the whole point of your PR is to move documentation generation into the main Renovate repository?

Do you need to change this comment, or is this still okay?

Reply:
currently it needs to be same for all files, rename later, see

https://github.com/renovatebot/renovate/blob/357484d594f178be2a50bf96c5f31ac6a7ffac5a/tools/docs/utils.ts#L5-L7
https://github.com/renovatebot/renovate/blob/357484d594f178be2a50bf96c5f31ac6a7ffac5a/tools/docs/utils.ts#L36-L49

the replace function is shared 😉